### PR TITLE
Organize pom's

### PR DESCRIPTION
**Pull request by danielpassos**

### CODE REVIEW

**Comment by danielpassos** [comment]
@secondsun :eyeglasses: ?

File: aerogear-android-authz-test/pom.xml [hunk 1]
```diff
@@ -30,9 +30,19 @@
...
         <mockito.version>1.9.5</mockito.version>
         <dexmaker.version>1.0</dexmaker.version>
         <dexmaker.mockito.version>1.0</dexmaker.mockito.version>
+        <android.support.test.version>0.3</android.support.test.version>
+
+        <!-- Plugins parameters -->
+        <android.maven.plugin.dex.xms>-Xms256m</android.maven.plugin.dex.xms>
+        <android.maven.plugin.dex.xmx>-Xmx2048m</android.maven.plugin.dex.xmx>
+
+        <maven.javadoc.skip>true</maven.javadoc.skip>
```
Comment (secondsun) [inline aerogear-android-authz-test/pom.xml]: What is this field used for?  I can't find it referenced in the poms

**Comment by secondsun** [comment]
Build passes, +1 (other than my question about the javadoc property)

File: aerogear-android-authz-test/pom.xml (hunk 1 above)
Comment (danielpassos) [inline aerogear-android-authz-test/pom.xml]: Just a copy and paste :disappointed: I'll remove here and at the another PR's

**Comment by danielpassos** [comment]
Done!

**Comment by secondsun** [comment]
landed

**Pull request merged**